partner_company_type/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Company Type',
+    'summary': 'Adds a company type to partner that are companies',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```

Comment:
@daramousk `website` should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)